cli/tests/it/cmd.rs

```diff
@@ -368,7 +368,7 @@ forgetest_init!(can_clean_config, |prj: TestProject, mut cmd: TestCommand| {
     cmd.assert_non_empty_stdout();
 
```

Comment:
didn't we just merge a PR that changed the format! s?

In any case, please undo because unrelated here.

Reply:
this was reverted like 10 commits ago , this isnt the current diff